lcls_tools/superconducting/scLinac.py

```diff
@@ -1696,7 +1696,8 @@ def __init__(
         stepper_class=StepperTuner,
         piezo_class=Piezo,
     ):
-        # type: (str, Linac, Type[Cavity], Type[Magnet], Type[Rack], bool, Type[SSA], Type[StepperTuner], Type[Piezo]) -> None
+        # _type: (str, Linac, Type[Cavity], Type[Magnet], Type[Rack],
```

Comment:
The underscore breaks the IDE typecasting. Was it included to be able to add the line break? I would prefer keeping the cast over making the line shorter

Reply:
I was trying to break the line so it'd fit within the 120 length. I didn't want to remove it entirely even though it's commented. Let me see if I can remove the _ and keep the line break :)

Reply:
Ah yeah it's a comment, but it's an IDE type hint. I tried putting them directly in the argument list (like, `piezo_class: Type[Piezo] = Piezo` but I got errors so I kept it this way

Reply:
Added a noqa tag next to this line so flake8 ignores line-length here